scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -300,6 +302,10 @@ trait TypedPipe[+T] extends Serializable {
   def mapValues[K, V, U](f: V => U)(implicit ev: T <:< (K, V)): TypedPipe[(K, U)] =
     raiseTo[(K, V)].map { case (k, v) => (k, f(v)) }
 
+  /** Similar to mapValues, but allows to return a collection of outputs for each input value */
+  def flatMapValues[K, V, U](f: V => TraversableOnce[U])(implicit ev: T <:< (K, V)): TypedPipe[(K, TraversableOnce[U])] =
```

Comment:
But if I use `fn(v1).map { v2 => k -> v2 }` This returns a Traversable[U] (or Traversable[V2] from example above)